pkg/scheduler/scheduler.go

```diff
@@ -349,14 +349,14 @@ func (sched *Scheduler) schedule(pod *v1.Pod, pluginContext *framework.PluginCon
 // preempt tries to create room for a pod that has failed to schedule, by preempting lower priority pods if possible.
 // If it succeeds, it adds the name of the node where preemption has happened to the pod spec.
 // It returns the node name and an error if any.
-func (sched *Scheduler) preempt(fwk framework.Framework, preemptor *v1.Pod, scheduleErr error) (string, error) {
+func (sched *Scheduler) preempt(pluginContext *framework.PluginContext, fwk framework.Framework, preemptor *v1.Pod, scheduleErr error) (string, error) {
```

Comment:
the framework should be available in `sched.Framework`. Pass `pluginContext` as second argument to for consistency with `schedule`

Reply:
Like the reply above, I actually prefer to update pass pluginContext as first argument with schedule.

Reply:
Sure :)

Reply:
FYI, this is being handled in #82072

Reply:
We should do this in a separate small PR because we don't know if #82072 will move forward or not, and if yes in what form, so it may take a little while to merge.